.user.ini for php-fpm

tessus opened the pull request:
PHP settings in .htaccess are ignored by php-fpm. They have to
be put in a file called .user.ini

see #5846

alecpl commented:
Could you add a comment on top of the file that describes its purpose and maybe with reference to http://php.net/manual/en/configuration.file.per-user.php?

tessus commented:
Sure, will do. I just haven't done it because there was no comment in `.htaccess` either. I mean a comment related to what the purpose of the file is.
Which brings me to my next point: there's no reason why you shouldn't add the IfModule test for php7 as well.
I'll grab late lunch and make a few changes afterwards (in about 60 minutes). You can always decide to cherry pick the commits. Or I'll create another PR for the .htaccess file.

tessus committed:
.user.ini for php-fpm (CGI/FastCGI SAPI)

This file is read and processed by the CGI/FastCGI SAPI
PHP settings in .htaccess are only available to the PHP Apache module
also see: http://php.net/manual/en/configuration.file.per-user.php

tessus commented:
Done. Added a description as requested.

I've also created a separate PR #5850 for the `.htaccess` file. There's no reason to have the user change the file manually, if it can be easily done by adding another `IfModule` test. No more manual interaction necessary.

alecpl commented:
.htaccess is old and known technology, while .user.ini is less common and the file name is configurable. That's why I think the educative comment is needed. Thanks.

tessus commented:
Yes, it is true that .htaccess has been around for a long time, but you should be aware that the filename of .htaccess is also configurable.
Anyway, it now has a description. :-)
